v2/sunshine/notely_timeline_app/assets/ticket_sidebar.html

```diff
@@ -63,14 +63,14 @@
         function fetchNotes() {
           var requesterId = ticket['ticket.requester.id'];
           var settings = {
-            url: '/api/cdp/events?user_id=' + requesterId,
+            url: '/api/v2/users/'+ requesterId + '/events?filter[source]=notely',
             type: 'GET',
             dataType: 'json',
           };
 
           client.request(settings).then(
             function(response) {
-              state.data = response.data.filter(event => event.type === 'call' || event.type === 'email' || event.type === 'event' || event.type === 'meeting' || event.type === 'note');
+              state.data = response.events.filter(event => event.type === 'call' || event.type === 'email' || event.type === 'event' || event.type === 'meeting' || event.type === 'note');
```

Comment:
If we're filtering on the source, seems unnecessary to have this additional filtering layer on the type, right?

Reply:
True! I have removed that filter line